addon/services/url.js

```diff
@@ -0,0 +1,18 @@
+import Service from '@ember/service';
+import { getOwner } from '@ember/application';
+
+export default class URLService extends Service {
+  get prefix() {
+    const config = getOwner(this).resolveRegistration('config:environment');
```

Comment:
I would prefer if you made use of https://www.npmjs.com/package/ember-get-config which we've pretty much standardised around in most empress projects (but it seems like we didn't in this one 🤔 )

Reply:
so getting config is probably not going to be the worst of our problems in Empress once Embroider lands 😂 😭  It looks like they have a way forward to get ember-get-config working effectively anyway so I'm happy to rely on it at this stage 👍